site/app/templates/GlobalHeader.twig

```diff
@@ -123,7 +123,7 @@
             </div>
             <div id="menu-overlay"></div>
             <nav class="flex-row shadow">
-                <div id="nav-links" style="position:relative; height:80px">
```

Comment:
These changes seem unrelated to this PR. They are also causing a small difference in display between master and this branch: the duck and Submitty are closer together on the top bar

On Master:
![image](https://user-images.githubusercontent.com/11095297/88560321-fad18f00-cffb-11ea-9f80-f1247e3d77d3.png)

On Branch:
![image](https://user-images.githubusercontent.com/11095297/88560487-2f454b00-cffc-11ea-953d-3e83c2bb3647.png)